packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.html

```diff
@@ -10,7 +10,7 @@
           <lightning-formatted-text value={labels.sortBy}></lightning-formatted-text>
         </lightning-layout-item>
         <lightning-layout-item>
-          <lightning-combobox name="sort" value={value} variant="label-hidden" label={labels.sortBy} placeholder={labels.relevancy} options={options} onchange={handleChange}>
+          <lightning-combobox class="sort__combobox" name="sort" value={value} variant="label-hidden" label={labels.sortBy} placeholder={labels.relevancy} options={options} onchange={handleChange}>
```

Comment:
I'm not sure how is this CSS class is overriding the Salesforce CSS class defined inside the lightning-combobox component since this class you  are adding is in a higher level than the SF class. are you sure it is overriding it?

Reply:
In another note I suggest trying the following solution to see if it works:

The `lightning-combobox` component has a public property called: `dropdown-alignment` it specifies where the drop-down list is aligned. One of the values that this property could take is `auto` which will let the component determine where to open the list based on space available.
So it's worth trying to see if this component by it self is able to adapt according to the SF boundaries.

ref: https://developer.salesforce.com/docs/component-library/bundle/lightning-combobox/specification

Reply:
1) Yes I checked it is overriding it.

2) I already tried that when I was experimenting with the potential idea to make it dynamically aligned-right and auto doesnt work :( It was still overflowing... I think it might not account properly for the side of the container or something

Reply:
Yea I would suggested replacing simply the `dropdown-alignment` to set it to `dropdown-alignment="right"` It makes more sense that the dropdown fits by being right aligned instead of limiting the width etc.